terraform/scripts/manually-upload-certs.sh

```diff
@@ -2,8 +2,8 @@
 
 set -eu
 
-export PASSWORD_STORE_DIR=${CERT_PASSWORD_STORE_DIR}
-
+export PASSWORD_STORE_DIR=~/.paas-pass-high
```

Comment:
The [credentials README]() documents to add the following to your `.bashrc`:
If you use Bash (probably), add this to your ~/.bashrc:
```sh
# Load the custom .*-pass I have
for i in ~/.*-pass; do
  [ -e $i/.load.bash ] && . $i/.load.bash
done
```
This is incompatible with this naming convention because it needs the directory to end `-pass`.

I think it would be better to still allow overriding this with an environment variable (a proper one as opposed to `make`'s approximation of them.).